content/guide/_index.md

```diff
@@ -8,145 +8,207 @@ Welcome! This guide will help you get started with a Hugo Theme, including how t
+```bash
+git add -A
+git commit -m "first commit of all website files"
+```
 
-#### Default Parameters
+### Step 7: Test your website
 
-Default parameters are those defined by the theme developer. Default parameters start with `[params]` heading.
+Hugo comes with an integrated development server. You can easily start the server with the following command:
 
- * `logo` is for your site logo, give the logo directory here (i.e. `logo = images/logo.png`). Logo supports `png`, `jpg`, `gif`, or `svg` format. If you leave it empty, then `title` (default configuration title) will replace the logo.
+```shell script
+hugo server
+```
 
- * `footer_logo` is for your footer logo. Most of the themes don't need that. It's only need when the footer logo is different color.
+Hugo will load everything and start the server and let you know at which URL you can load the test site. Open your browser and enjoy your new shiny Hugo website.
 
- * `favicon` is defined in your theme header. So you don't need to specify the directory here. Just put your favicon in `static/images` folder. Remind that you need to put this favicon name `favicon.png`. Otherwise, it won't load. If the theme already has a favicon, then replace it with your new one. It will be shown on the browser tab when you compiled the theme again. If it's not showing, then you need to clear your browser cache and reload again.
+## Local development in a nutshell
 
- * `description` is for your default meta description, it will show on your homepage and other pages that have not set any meta description on this page front-matter.
+{{< youtube dyZcRRWiuuw >}}
 
- * `author` is also a meta tag. It belongs to the site author.
+**If the list above sounds like a foreign language to you, that’s okay! Let’s break it down:**
 
- * `mobile` is your default mobile number. If your theme has an option to make a call, then you can find it on default parameters.
+* **`Hugo` is a static site generator**. If you don’t have Hugo installed, you’ll need to [download](https://github.com/gohugoio/hugo/releases) and [install](https://gohugo.io/getting-started/installing/) it [make sure you have downloaded the [Hugo Extended](#install-hugo-extended-version) version for compiling scss files].
+* **A local server** is a temporary server environment on your computer-generated by your theme’s build tool. This avoids browser limitations of serving files directly from your computer’s filesystem (i.e. [CORS](https://developer.mozilla.org/en-US/docs/Glossary/CORS) limitations).
+* **Live reload** is a convenience function that many themes will include as part of their build tooling. It simply means that when you edit source files after they are recompiled, your browser will magically refresh and display the changes without a manual refresh.
```

Comment:
Codacy found an issue: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/gh/gethugothemes/gethugothemes-docs/pullRequest?prid=6304190)